### PR TITLE
Fix - tracking Application Insights telemetry

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #71

### CODE REVIEW

File: src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs [hunk 1]
```diff
@@ -103,7 +103,7 @@ public void LogRequest_SinksToApplicationInsights_ResultsInRequestTelemetry()
                 ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
 
                 HttpMethod httpMethod = GenerateHttpMethod();
-                string host = _bogusGenerator.Company.CompanyName();
+                string host = _bogusGenerator.Company.CompanyName().Replace(" ", "");
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: Why is this required? Does it cause any issues?

Comment (stijnmoreels) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: Yes, spaces in host names are now allowed.

Comment (tomkerkhove) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: Let's enforce it in our extension then

Comment (stijnmoreels) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: The extension already uses `HttpRequest` instead so that's already covered, right? The tests only uses this for the stubbing.

Comment (tomkerkhove) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: Does HttpRequest throw an exception for that host name or not? If not, we should add a check in our extension to enforce it maybe?

Comment (stijnmoreels) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: I don't think it guards it very well, so I'll add some extra guards here.

Comment (tomkerkhove) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:106]: Ok thanks!
